### PR TITLE
Fix double delete on backspace on Android

### PR DESCRIPTION
**Pull request by gspencergoog**
## Description

This disables the delete handling on the engine side, since now that we send events to the framework first, the framework is handling it.

## Related Issues

 - Fixes https://github.com/flutter/flutter/issues/70744

## Tests

 - Removed engine tests for "delete backwards" because the framework is testing those.

## Breaking Change

- [X] No, no existing tests failed, so this is *not* a breaking change.

### CODE REVIEW

**Comment by yangyxd** [comment]
Waiting for merge code

File: shell/platform/android/test/io/flutter/plugin/editing/InputConnectionAdaptorTest.java [hunk 1]
```diff
@@ -1028,169 +1028,6 @@ public void testCursorAnchorInfo() {
     assertNull(testImm.lastCursorAnchorInfo);
   }
 
-  @Test
```
Comment (goderbauer) [inline shell/platform/android/test/io/flutter/plugin/editing/InputConnectionAdaptorTest.java:1031]: Should we add a test to verify that KeyEvent.KEYCODE_DEL doesn't do anything?

**Review by goderbauer — APPROVED** [review]
LGTM

File: shell/platform/android/test/io/flutter/plugin/editing/InputConnectionAdaptorTest.java (hunk 1 above)
Comment (gspencergoog) [reply shell/platform/android/test/io/flutter/plugin/editing/InputConnectionAdaptorTest.java:1031]: OK, I'll add that. Should be a simple matter of reversing the expects in the original test.

**Pull request merged**